### PR TITLE
Update copy on About page

### PR DESCRIPTION
**Pull request by littleforest**
[Asana ticket](https://app.asana.com/0/1203289004376659/1206398384307493/f)

Screenshot 2024-02-08 at 9 22 16 AM

### CODE REVIEW

File: app/views/pages/about.html.erb [hunk 1]
```diff
@@ -7,16 +7,32 @@
...
+      <p class="mb-1">The Library can help you:</p>
+      <ul class="list-disc list-inside" style="list-style-type: disc;">
+        <li class="mb-1.5"><strong><em>Search by industry, occupation, state, and more</em></strong> to find the apprenticeship program training standards most helpful for you.</li>
+        <li class="mb-1.5"><strong><em>Find multiple examples of program standards for one occupation</em></strong> to provide options and ideas for your own apprenticeship.</li>
+        <li class="mb-1.5"><strong><em>Access approved work process schedules and related instruction outlines</em></strong> you can use as a starting point.</li>
+        <li><strong><em>Build from existing program standards</em></strong> to help you more quickly design your apprenticeship program.</li>
+      </ul>
+
+      <h4 class="font-bold text-xl mt-6 mb-2">Additional Resources</h4>
+      <ul class="list-disc list-inside" style="list-style-type: disc;">
```
Comment (littleforest) [inline app/views/pages/about.html.erb:30]: @jenlu33 @FerPerales  (@fatimaburke ?) I cannot get the tailwind `list-disc` to work (I hard-coded a `style` here). `list-decimal` works without issue.

File: app/views/pages/about.html.erb [hunk 2]
```diff
@@ -7,16 +7,32 @@
...
+        <li class="mb-2">Read the <%= link_to "fact sheet", asset_path("documents/Urban-FACTSHEET-RASL.pdf", skip_pipeline: true), class: "underline underline-offset-4", style: "text-underline-offset: 4px" %> to learn more about the Registered Apprenticeship Standards Library.</li>
```
Comment (littleforest) [inline app/views/pages/about.html.erb:31]: @jenlu33 @FerPerales  (@fatimaburke ?) also cannot get the tailwind `underline-offset-4` to work (I hard-coded a `style` here). Any ideas?

**Comment by FerPerales** [comment]
@littleforest my first guess is that the tailwind version we are using is different from the docs you are reading

**Comment by littleforest** [comment]
> my first guess is that the tailwind version we are using is different from the docs you are reading

We're on the latest `tailwindrails-css` version and it is using Tailwind 3.4.1 :( https://github.com/rails/tailwindcss-rails/commit/f989541ccfd8029fd0fdc967bb50ee6d62e8b51c

**Comment by littleforest** [comment]
@FerPerales @jenlu33 please review and I will make a follow-up ticket re the Tailwind issues.

**Review by FerPerales — APPROVED** [review]

**Pull request merged**